### ISSUE TITLE
I got an issue while training the model 

### ISSUE THREAD

**Issue by bitrookie** [polimi-ispl/icpr2020dfdc#15]
Hi, 

Sorry for bothering. When I tried to train the module with dfdc dataset. The following error is reported. May I ask whether I missed anything.
![image](https://user-images.githubusercontent.com/37587490/107858289-7ac32780-6e6e-11eb-8c86-7c4a53b3439a.png)

regards,

**Comment by CrohnEngineer** [comment]
Hey @bitrookie ,

I see you have closed the issue, did you find the solution to your problem?
If you have any other question just let us know.
Bests,

Edoardo

**Comment by bitrookie** [comment]
Hello @CrohnEngineer ,

Yes, I have found the reason.
May I ask how long did it take for  you to train the model and what gpu or TPu are you using since it takes me quite long for even running one epoch.

bitrookie

**Comment by CrohnEngineer** [comment]
Hey @bitrookie ,

we trained the networks on a machine equipped with an Intel Xeon E5-2687W-v4 and a NVIDIA Titan V.
I sincerely don't remember how long it took for the networks to be ready (it might have taken an entire day, but I'm not sure), however we never trained them for an entire epoch as they usually overfitted a lot earlier.
Indeed, if you look at the paper and at the code, we perform validation and checks on the validation loss based on the number of batch iterations rather than the number of epochs.
So, in conclusion, don't be scared if you are taking some time, the datasets are pretty huge :)

Edoardo

**Comment by hzhumsc** [comment]
Hi, @bitrookie ,

I am having the same issue here when I try to train the EfficientNetAutoAttB4 model with Celebdf. May I ask how you solved your problem earlier?

Best regards,
